diff --git a/cmake/onnxruntime_providers_cuda.cmake b/cmake/onnxruntime_providers_cuda.cmake
--- a/cmake/onnxruntime_providers_cuda.cmake
+++ b/cmake/onnxruntime_providers_cuda.cmake
@@ -320,6 +320,7 @@
           shorten-64-to-32
           unneeded-internal-declaration
           unknown-warning-option
           unused-private-field
+          unused-function
           unused-variable)
           target_compile_options(${target} PRIVATE "$<$<NOT:$<COMPILE_LANGUAGE:CUDA>>:-Wno-error=${CLANG_WARNING}>")
         endforeach()
@@ -337,6 +338,7 @@
             shorten-64-to-32
             unneeded-internal-declaration
             unknown-warning-option
             unused-private-field
+            unused-function
             unused-variable)
             target_compile_options(${target} PRIVATE "$<$<COMPILE_LANGUAGE:CUDA>:SHELL:-Xcompiler -Wno-error=${CLANG_WARNING}>")
           endforeach()
